activerecord/lib/active_record/core.rb

```diff
@@ -254,11 +254,19 @@ def self.connected_to_stack # :nodoc:
         end
       end
 
+      def self.connection_class=(b) # :nodoc:
+        @connection_class = b
+      end
+
+      def self.connection_class? # :nodoc:
+        @connection_class
+      end
+
       def self.abstract_base_class # :nodoc:
```

Comment:
Isn't this "abstract_base_class" a proper name anymore?

```suggestion
      def self.connection_base_class # :nodoc:
```

Reply:
Thanks for the reminder, I fixed this 👍🏻